apps/core/components/header/cart.tsx

```diff
@@ -17,24 +18,19 @@ export const CartLink = ({ children }: { children: ReactNode }) => (
 
 export const Cart = async () => {
   const cartId = cookies().get('cartId')?.value;
+  let count: number | undefined;
 
-  if (!cartId) {
-    return (
-      <CartLink>
```

Comment:
I'm assuming there was no particular reason we needed to exclude 

`<span className="sr-only">Cart Items</span>` 

when there was no cart, but correct me if I'm wrong here.

Reply:
For accessibility we wanted an empty cart to just say `"Link, cart"` and a cart with items, `"Link, Cart Items {count}"` for screen readers.

Reply:
Ah gotcha, I'll see about supporting that